scripts/README.md

```diff
@@ -0,0 +1,11 @@
+# UniOne API test script
+Usage:
+
+
+```bash
+compose require unione/unione-php
+cd vendor/unione/unione-php && composer test {YOUR-HOST-NAME} {YOUR-API-KEY}
+```
+You should copy example.config.php to config.php file and set your parameters.
+
+After that, specify that it returns 0 on success and 1 in case of failure.
```

Comment:
@tymoshchuk-o it was a comment for you :) Please correct the description of exit codes.